services/indices/helpers.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import Joi from 'joi';
+import InternalError from '@/errors/InternalError';
+
+export const buildUrl = {
+  simple: (code) =>
+    `https://api.bcb.gov.br/dados/serie/bcdata.sgs.${code}/dados`,
+  lastRecords: (code, numberOfRecords) =>
+    `https://api.bcb.gov.br/dados/serie/bcdata.sgs.${code}/dados/ultimos/${numberOfRecords}`,
+};
+
+const defaultParams = {
+  formato: 'json',
+};
+
+export const fetchData = async (url, params = {}, schema) => {
```

Comment:
O sonarcloud indicou isso como um code smell. Seria interessante trocar a ordem desses parâmetros e consequentemente os locais que o chamam

![image](https://user-images.githubusercontent.com/16273730/224484809-fbdd1b97-6c8f-4804-bb15-116c48b51fcd.png)

```suggestion
export const fetchData = async (url, schema,  params = {}) => {
```

Reply:
Show! Feito a correção sugerida